tests/e2e_operations.py

```diff
@@ -78,39 +98,66 @@ def test_forced_ledger_chunk(network, args):
     # Issue some more transactions
     network.txs.issue(network, number_txs=13)
 
-    # Wait for the signature interval to ensure we see at least one signature
-    time.sleep(args.sig_ms_interval / 1000)
-
     ledger_dirs = primary.remote.ledger_paths()
 
     # Check that there is indeed a ledger chunk that ends at the
     # first signature after proposal.completed_seqno
     ledger = ccf.ledger.Ledger(ledger_dirs)
-    for chunk in ledger:
-        first = last = next_signature = None
-        for tx in chunk:
-            pd = tx.get_public_domain()
-            if first is None:
-                first = pd.get_seqno()
-            else:
-                last = pd.get_seqno()
-            tables = pd.get_tables()
-            if (
-                pd.get_seqno() >= proposal.completed_seqno
-                and next_signature is None
-                and ccf.ledger.SIGNATURE_TX_TABLE_NAME in tables
-            ):
-                next_signature = pd.get_seqno()
-        if first <= proposal.completed_seqno and proposal.completed_seqno <= last:
-            LOG.info(
-                f"Found ledger chunk {chunk.filename()} with chunking proposal @{proposal.completed_seqno} and signature @{next_signature}"
-            )
-            assert last == next_signature
-            assert next_signature - proposal.completed_seqno < args.sig_tx_interval
-
+    chunk, _, last, next_signature = find_ledger_chunk(ledger, proposal.completed_seqno)
+    LOG.info(
+        f"Found ledger chunk {chunk.filename()} with chunking proposal @{proposal.completed_seqno} and signature @{next_signature}"
+    )
+    assert chunk.is_complete and chunk.is_committed()
+    assert last == next_signature
+    assert next_signature - proposal.completed_seqno < args.sig_tx_interval
     return network
 
 
+@reqs.description("Forced snapshot")
+def test_forced_snapshot(network, args):
+    primary, _ = network.find_primary()
+
+    # Submit some dummy transactions
+    network.txs.issue(network, number_txs=7)
```

Comment:
The numbers themselves don't matter, but I wanted some more unrelated transaction on the ledger just in case. And it did help find problems with other tests...